DependencyInjection/PrestaSitemapExtension.php

```diff
@@ -39,6 +39,10 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter($this->getAlias() . '.defaults', $config['defaults']);
         $container->setParameter($this->getAlias() . '.default_section', $config['default_section']);
 
+        if ($this->isConfigEnabled($container, $config['alternate'])) {
```

Comment:
Otherwhise, the parameter is not set, which will cause errors : missing parameter : https://travis-ci.org/prestaconcept/PrestaSitemapBundle/builds/592629655

Reply:
fixed

Reply:
Not sure it is fixed, when the config is disabled (`enabled: false`), the parameter doesn't exist too and the container compilation fails. Maybe set `presta_sitemap.alternate` as an empty array when the section is disabled ?

Reply:
We should add a use case for this.
Running this bundle with that feature disabled should be tested IMO

Reply:
I'll try to create a use case and a test soon